src/AvionicsDownlink.cpp

```diff
@@ -110,11 +110,9 @@ int16_t Avionics::compressData() {
     lengthBits += compressVariable(data.RE_ARM_CONSTANT,             0,    4,        8,  lengthBits);
     lengthBits += compressVariable(data.VALVE_ALT_LAST,             -2000, 50000,    11, lengthBits);
     lengthBits += compressVariable(data.BALLAST_ALT_LAST,           -2000, 50000,    11, lengthBits);
-    lengthBits += compressVariable(data.SPAG_STATE.effort*1000,            -2, 2, 12, lengthBits);
-    lengthBits += compressVariable(data.SPAG2_STATE.effort*1000,           -2, 2, 12, lengthBits);
     lengthBits += compressVariable(data.LAS_STATE.v,           -10,     10,   11, lengthBits);
```

Comment:
These could probably use comments so that HABMC parses it more cleanly. This will get parsed as `las_state.v` -- if we added a comment saying something like `// Lasagna velocity` it will get parsed as `lasagna_velocity`, which imo is easier to understand.

Reply:
ok, good to know